release/models/devices-manifest/openconfig-terminal-device-property-types.yang

```diff
@@ -0,0 +1,459 @@
+  }
+
+  identity CWDM {
+    base  GRID_TYPE;
+    description
+      "Fixed frequency grid as specified in  ITU-T G.694.1 (10/2020).";
+  }
+
+  identity FLEX {
+    base  GRID_TYPE;
+    description
+      "Flexible frequency grid as specified in  ITU-T G.694.1 (10/2020).";
+  }
+
+  identity GRIDLESS {
+    base  GRID_TYPE;
+    description
+      "No grid";
+  }
+
+  identity UNSPECIFIED {
+    base  GRID_TYPE;
+    description
+      "Unspecified/proprietary frequency grid";
+  }
+
+  identity ADJUSTMENT_GRANULARITY {
+    description
+      "Base identity for the adjustment granularity or nominal central
+      frequency granularity, according to  ITU-T G.694.1 (10/2020) standard,
+      defined as the minimun spectrum separation between the central
+      frequencies of two contiguous optical channels";
+  }
+
+  identity G_100GHZ {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value of 100 gigahertz.";
+  }
+
+  identity G_50GHZ {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value of 50 gigahertz.";
+  }
+
+  identity G_25GHZ {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value of 25 gigahertz.";
+  }
+
+  identity G_12_5GHZ {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value of 12.5 gigahertz.";
+  }
+
+  identity G_6_25GHZ {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value of 6.25 gigahertz.";
+  }
+
+  identity G_3_125GHZ {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value of 3.125 gigahertz.";
+  }
+
+  identity UNCONSTRAINED {
+      base  ADJUSTMENT_GRANULARITY;
+      description
+        "Adjustment granularity value unconstrained, i.e.,
+        no minimun spacing between channels is defined.";
+  }
+
+
+  identity FEC {
+    description
+      "Forward Error Correction base identity.";
+  }
+
+  identity FEC_HD {
+    base FEC;
+    description
+      "Hard-Decision (HD) Staircase FEC, defined in
+      ITU-T G.709.2 (07/18)";
+  }
+
+  identity FEC_G {
+    base FEC;
+    description
+      "Generic FEC, Reed Solomon (255,239) coding schema,
+      defined in ITU-T G.975 (10/2000).";
+  }
+  identity FEC_E {
+    base FEC;
+    description
+      "Enhanced FEC (EFEC) from G.975.1 Clause I.4.";
+  }
+  identity FEC_O {
+    base FEC;
+    description
+      "OpenROADM 200G Open FEC (oFEC) soft-decision coding
+      schema.";
+  }
+
+  identity FEC_C {
+    base FEC;
+    description
+      "OIF 400G concatenated FEC (cFEC) with soft-decision inner
+       Hamming code and hard-decision outer Staircase code";
+  }
+
+  identity FEC_OTHER {
+    base FEC;
+    description
+      "Placeholder identity to refer to any other propiertary or
+      non-propiertary FEC coding schema non-defined in this model
+      yet.";
+  }
+
+  identity MODULATION_FORMAT {
```

Comment:
Do you want to include a "proprietary" or "other" here as I am seeing most of the newer equipment is all proprietary these days?

Reply:
Agree. Updated in the latest commit.